### DISCUSSION TITLE
How to disable rotation but keep zoom?

### DISCUSSION THREAD

**Discussion by berryhijwegen** [uber/react-map-gl#2039] [Q&A]
I have been trying to disable the rotation, but enable the zoom feature. It seems those two are linked together in the Map component by the `touchZoomRotate` attribute. Is it possible to disable/enable them separately?

**Comment by Pessimistress** [comment]
```jsx 
<Map
  initialViewState={...}
  touchZoomRotate={true}
  bearing={0}
/>
